Basic-Video-Chat/Basic-Video-Chat.xcodeproj/project.pbxproj

```diff
@@ -311,7 +312,6 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
```

Comment:
Why did you remove `LaunchImage`?

Reply:
I believe that `LaunchImage` has been deprecated. It was throwing an error/warning. The file in the assets folder was also empty so I got rid of it.

Reply:
👍